docs/source/ephys_at_swc/getting_started.md

```diff
@@ -0,0 +1,133 @@
+contains a clearly written overview of preprocessing steps. Similarly, Bill Karsh's
+[guide](https://billkarsh.github.io/SpikeGLX/help/catgt_tshift/catgt_tshift/) on the
+SpikeGLX website also gives a useful overview.
+
+[This paper](https://www.cell.com/neuron/pdf/S0896-6273(19)30174-6.pdf) provides a more
+technical treatment of digital filtering, a key step in preprocessing and analysis.
+
+### Spike Sorting
+
+[This video](https://www.youtube.com/watch?v=vSydfDvsewY) with Christophe Pouzat, provides
+an excellent overview of the spike-sorting problem.
+
+[This chapter](https://neurophysics.ucsd.edu/publications/obd_ch3_2.pdf) provides a more
+technical overview of spike sorting. It is also recommended to check out the
+papers of existing spike sorting algorithms. A list of the main
+spike sorters can be found [here](https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html#supported-spike-sorters).
+
+
+### Quality Metrics and Manual Curation
+
+Assessing the quality of spike-sorting is a key to producing high-quality data.
+
+These two papers provide a nice introduction to quality metrics of spike sorting:
+
+[Quality Metrics to Accompany Spike Sorting of Extracellular Signals](https://www.jneurosci.org/content/31/24/8699)
+
+[Improving data quality in neuronal population recordings](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5244825/)
+
+[Phy](https://github.com/cortex-lab/phy)
+is the most popular tool for performing manual curation of spike sorting results.
+A great guide by [Steve Lenzi](https://phy.readthedocs.io/en/latest/sorting_user_guide/) takes you
+through the key steps for manual curation.
+
+More recently, advances in the automation of curation has been made using the
+[Bombcell package](https://github.com/Julie-Fabre/bombcell).
+
+SpikeInterface also maintain a set of quality metrics,
+[explained in detail](https://spikeinterface.readthedocs.io/en/latest/modules/qualitymetrics.html)
+in their documentation.
+
+## SpikeInterface
+
+Visit the SpikeInterface
+[GitHub](https://github.com/SpikeInterface/spikeinterface)
+and
+[Documentation](https://spikeinterface.readthedocs.io/en/latest/index.html)
+to get started. Note their documentation currently points to the developer
+version, select your installed version from the list at the bottom-left
+of the page.
+
+## Other Community Tools
+
+### Analysis
+
+SpikeInterface
+is mainly focused on preprocessing, spike sorting and quality metrics.
+The [Pynapple](https://github.com/pynapple-org/pynapple)
+and
+[Elephant]( https://neuralensemble.org/elephant/)
+both provide nice toolboxes for analysing data post-sorting.
+
+The [SpikeForest](https://spikeforest.flatironinstitute.org/)
+project is an excellent resource for assessing the performance of
+different spike-sorting algorithms across probe types and brain regions.
+
+### Pipelines
+
+While the community has gradually coalesced around SpikeInterface,
```

Comment:
Is this actually true (vs should be)? I rarely hear researchers mention it.

Reply:
Hey I don't think this is a good way to word it. Basically on the allen pipeline, there is a warning saying they recommend using SI instead, and Olivier from IBL has said the only reason they are not using SI is because they predate it. But it's true this is not generally the case. Do you think best approach is to put these points in brackets next to the links to relevant repos? or not mention at all?

Reply:
> on the allen pipeline, there is a warning saying they recommend using SI instead, and Olivier from IBL has said the only reason they are not using SI is because they predate it.

This is good to hear though!